ledger/src/bigtable_upload.rs

```diff
@@ -127,8 +130,11 @@ pub async fn upload_confirmed_blocks(
     };
 
     if blocks_to_upload.is_empty() {
-        info!("No blocks need to be uploaded to bigtable");
-        return Ok(*last_blockstore_slot);
+        info!(
+            "No blocks between {} and {} need to be uploaded to bigtable",
```

Comment:
Same as above - if we have multiple running async, would be nice to know the range where there was no upload needed